.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
                         repo: context.repo.repo,
                         issue_number: context.issue.number,
                       }).then(({data}) => {
-                        const priorComment = data.find(comment => comment.body.startsWith('# Branch Preview'));
+                        const priorComment = data.find(comment => comment.body.startsWith('## Branch preview') || comment.body.startsWith('# Branch Preview'));
```

Comment:
This messages out the older version from existing PRs by replacing them or the new versions with the updated comment.